modules/nf-core/variancepartition/dream/main.nf

```diff
@@ -8,7 +8,7 @@ process VARIANCEPARTITION_DREAM {
         'community.wave.seqera.io/library/bioconductor-edger_bioconductor-variancepartition_r-optparse:ba778938d72f30c5' }"
 
     input:
-    tuple val(meta), val(contrast_variable), val(reference), val(target), val(formula)
+    tuple val(meta), val(contrast_variable), val(reference), val(target), val(formula), val(comparison)
```

Comment:
Convention in nf-core is for non-mandatory non-file things to go in via ext.args. Otherwise people end up having to supply '[]' or '' or other messy things.

Reply:
Doing that would also mean you don't need to change the workflow logic, you can pass it from the meta when it's available.

Reply:
Yes, I suppose that's true. But it might be simpler to leave those in place for now and resolve separately. I'm mostly concerned with not adding additional subworkflow complexity for this.

Reply:
@atrigila @grst I chatted with the maintainers, and the consensus was that in a situation where e.g. A + B OR C + D were required, a sensible thing might be to have two separate input channels. But this is not the case here, since these can be somewhat overlapping (right)?

Let's keep your version for now, I hadn't considered this point of Gregor.

Reply:
In a future version they'll be mutually exclusive. But sounds good to keep it as is for the current iteration.